http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/config/KubernetesAppConfig.scala

```diff
@@ -81,7 +81,6 @@ final case class CoaAppConfig(chartName: ChartName,
     .collect {
       case ServiceName("cbas")     => Cbas
       case ServiceName("cbas-ui")  => CbasUI
-      case ServiceName("wds")      => Wds
```

Comment:
Yeah I think the `Wds` case object could just be removed entirely too. The intent of `CoaService` was to enumerate the individual services _within_ cromwell-on-azure… and WDS won't be part of CoA anymore.